src/Repec.php

```diff
@@ -554,6 +567,8 @@ public function getEntityTemplate(ContentEntityInterface $entity) {
       }
     }
 
+    $this->moduleHandler->alter("repec_{$series_type}_template", $template, $entity);
```

Comment:
There's a case here for there being a generic alter, like `repec_template`, that allows a module to work on all the templates at once. Is that feasible?

Reply:
It is feasible. But there could be a case when the mapping will vary by series types.

For example, `wpaper` chooses author data to fetch from `prop_1`, while `journl` chooses to fetch the data from `prop_2`. In that case it will be a long generic hook handling all the cases, vs. small hooks handling the cases separately. @mpetrovic

Reply:
There could be both.

Reply:
There is a single hook now.